tests/external_C/test.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
```

Comment:
Could you rename the folder with the flag, i.e., extern-so or something like that?